Observables/README.md

```diff
@@ -0,0 +1,3 @@
+# Observables
+
+A description of this package.
```

Comment:
⚠️  Would you mind updating this to:

```md
These classes are our own implementation of [ReactiveX observables](http://reactivex.io/) that we used when we were targeting iOS 12+. But now that our minimum iOS version is iOS 13, we should stop using them and use [Combine](https://developer.apple.com/documentation/combine) instead. 

We should also migrate the existing usages of these classes to use Combine. The classes were intentionally created to be compatible with both ReactiveX and Combine so the migration should be smooth-ish.
```

🙂

Reply:
Well that answers that question – I was wondering why we had this given iOS 13  😁

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/3305/commits/b11a4683399736a08f5333cae7447d4af101fe4b

Reply:
Thanks! 😄